plugins-list.yaml

```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
 plugins/auth-backend-module-microsoft-provider:
 plugins/auth-backend-module-oauth2-provider:
 plugins/auth-backend-module-oauth2-proxy-provider:
-plugins/auth-backend-module-oidc-provider:
+# plugins/auth-backend-module-oidc-provider: # Included as static plugin in RHDH
 plugins/auth-backend-module-okta-provider:
+plugins/auth-backend-module-onelogin-provider:
 plugins/auth-backend-module-pinniped-provider:
 plugins/auth-backend-module-vmware-cloud-provider:
-#plugins/catalog: ==> Added as a static plugin in RHDH
-#plugins/catalog-backend: ==> Added as a static plugin in RHDH
+plugins/auth-node:
+plugins/auth-react:
+plugins/bitbucket-cloud-common:
+# plugins/catalog  ==> Added as a static plugin in RHDH
+# plugins/catalog-backend  ==> Added as a static plugin in RHDH
 plugins/catalog-backend-module-aws:
 plugins/catalog-backend-module-azure:
 plugins/catalog-backend-module-backstage-openapi:
-plugins/catalog-backend-module-bitbucket-cloud:
-plugins/catalog-backend-module-bitbucket-server:
+plugins/catalog-backend-module-bitbucket:
+# plugins/catalog-backend-module-bitbucket-cloud  ==> wrapper
+# plugins/catalog-backend-module-bitbucket-server  ==> wrapper
 plugins/catalog-backend-module-gcp:
 plugins/catalog-backend-module-gerrit:
-plugins/catalog-backend-module-github-org:
-plugins/catalog-backend-module-github:
-plugins/catalog-backend-module-gitlab:
+# plugins/catalog-backend-module-github  ==> wrapper
+# plugins/catalog-backend-module-github-org  ==> wrapper
+# plugins/catalog-backend-module-gitlab  ==> wrapper
+# plugins/catalog-backend-module-gitlab-org  ==> wrapper
 plugins/catalog-backend-module-incremental-ingestion:
-#plugins/catalog-backend-module-ldap: ==> New backend system support missing
-plugins/catalog-backend-module-msgraph:
+# plugins/catalog-backend-module-ldap  ==> wrapper
+# plugins/catalog-backend-module-logs  ==> wrapper
+# plugins/catalog-backend-module-msgraph  ==> wrapper
 plugins/catalog-backend-module-openapi:
 plugins/catalog-backend-module-puppetdb:
 plugins/catalog-backend-module-scaffolder-entity-model:
 plugins/catalog-backend-module-unprocessed:
-#plugins/catalog-graph: ==> Added as a static plugin in RHDH 
-#plugins/catalog-import: ==> Added as a static plugin in RHDH
+plugins/catalog-common:
+# plugins/catalog-graph ==> Added as a static plugin in RHDH :
+# plugins/catalog-import ==> Added as a static plugin in RHDH
+plugins/catalog-node:
+plugins/catalog-react:
```

Comment:
React package, not a plugin

Reply:
You mean the ones I commented or the `catalog-node` and `catalog-react` parts?